README.md

```diff
@@ -91,8 +91,8 @@ For you tweakers out there, here's some levers to pull:
 * **maxIpRate** (default: `0`) - Maximum rate if lag is <= minLag.
 * **maxIpRateHostViolation** (default: `0`) - Maximum rate if target host is currently exceeding the
 * configured `maxHostRatio`. This can be used to increase IP throttling if a particular host is being
-* targeted by a large number of IPs. Requests hitting this max rate will return `badIp` instead of
-* `hostViolation`.
+* targeted by a large number of IPs. Requests hitting this max rate will receive `hostViolation` while

```

Comment:
please fix these lines. They should not include `*` prefix unless they are unique options